### PR TITLE
V3.5 nx

### PR DESCRIPTION
**Pull request by oahcy**
add some macro for nx platform。

### CODE REVIEW

File: sources/tbb/machine/linux_ia64.h [hunk 1]
```diff
@@ -21,7 +21,9 @@
 #define __TBB_machine_linux_ia64_H
 
 #include <stdint.h>
-#include <ia64intrin.h>
+#if (CC_PLATFORM != CC_PLATFORM_NX)
+    #include <ia64intrin.h>
+#endif
```
Comment (minggo) [inline sources/tbb/machine/linux_ia64.h]: I don't think we should modify tbb source code, just not use tbb on nx.

File: sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp [hunk 2]
```diff
@@ -139,7 +139,9 @@
 
 uint16 pvmp3_decode_huff_cw_tab0(tmp3Bits *pMainData)
 {
+#if (CC_PLATFORM != CC_PLATFORM_NX && CC_PLATFORM != CC_PLATFORM_NX_WINDOWS)
     OSCL_UNUSED_ARG(pMainData);
+#endif
```
Comment (minggo) [inline sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp]: Why need to modify it?

File: sources/tbb/machine/linux_ia64.h (hunk 1 above)
Comment (oahcy) [reply sources/tbb/machine/linux_ia64.h]: yes i will remove it
I used it when there was no nvn using vukan before. Now things have changed.

File: sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp (hunk 2 above)
Comment (oahcy) [reply sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp]: this macro defined in a temp .h file

when project in chinese dir, this auto generate command line will no use on nx :
image

there ara no problem if use english path.

Comment (oahcy) [reply sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp]: I tried it again, it is a problem with the windows cl.exe compiler, which does not support the -include parameter, so this macro is not defined.

image

**Pull request merged**